packages/core/upload/admin/src/translations/en.json

```diff
@@ -38,7 +38,7 @@
   "list.assets-empty.search": "No result found",
   "list.assets-empty.subtitle": "Add one to the list.",
   "list.assets-empty.title": "There are no assets yet",
-  "list.assets-empty.title-withSearch": "There are no assets with the applied filters",
+  "list.assets-empty.title-withSearch": "There are no elements with the applied filters",
```

Comment:
Mh, we should probably check with @maevalienard to check on what she thinks a good default text should be used.